xc-admin/packages/xc-admin-common/src/governance_payload/index.ts

```diff
@@ -0,0 +1,105 @@
+import { ChainId, ChainName, toChainName } from "@certusone/wormhole-sdk";
+import * as BufferLayout from "@solana/buffer-layout";
+
+export declare const Action: {
+  readonly ExecutePostedVaa: 0;
+  readonly UpgradeContract: 0;
+  readonly AuthorizeGovernanceDataSourceTransfer: 1;
+  readonly SetDataSources: 2;
+  readonly SetFee: 3;
+  readonly SetValidPeriod: 4;
+  readonly RequestGovernanceDataSourceTransfer: 5;
+};
+
```

Comment:
FYI @Reisen is looking into whether we can use something like protobuf to define the binary format in one place and code gen all of this deserialization stuff.

Reply:
perfect